src/auto-evo/mutation_strategy/RemoveOrganelle.cs

```diff
@@ -57,22 +57,12 @@ public static RemoveOrganelle ThatCreateCompound(string compoundName)
 
         foreach (var organelle in organelles)
         {
-            bool forceBacteria = false;
-
             if (organelle.Definition == Nucleus)
```

Comment:
I think a comment here explaining the reasoning why nucleus is treated specially would be nice.